notebooks/shield-dash-import.py

```diff
@@ -0,0 +1,93 @@
+import datetime
+from operator import itemgetter
+
+import ujson
+from moztelemetry import get_pings
+
+
+PING_NAME = 'x-shield-studies'
+HEARTBEAT_NAME = 'x-shield-study-performance-1'
+STUDY_NAME = 'screen Performance X1'
+STUDY_START = '20160325'
+TODAY = datetime.date.today().strftime('%Y%m%d')
+
+
+def getShieldProps(p):
+    out = {
+        'client_id': p.get('clientId'),
+        'channel': p['application']['channel'],
+        'creation_date': datetime.datetime.utcfromtimestamp(
+            long(p['meta']['creationTimestamp']) // 1e9),
+        'doc_type': p['meta']['docType']
+    }
+    for k in ['firstrun', 'msg', 'name', 'variation']:
+        if k == 'firstrun':
+            out[k] = datetime.datetime.utcfromtimestamp(
+                int(p['payload'][k]) // 1e3)
+        else:
+            out[k] = p['payload'][k]
+    return out
+
+
+kwargs = {
+    'doc_type': 'OTHER',
+    'submission_date': (STUDY_START, TODAY),
+    'app': 'Firefox',
+}
+
+
+pings = get_pings(sc, channel='release', **kwargs).union(
+    get_pings(sc, channel='aurora', **kwargs)).union(
+        get_pings(sc, channel='beta', **kwargs)).union(
+            get_pings(sc, channel='nightly', **kwargs))
```

Comment:
Since RDDs are lazy you could also get pings for each channel and then unite them with `sc.union` like

```python
channels ['release', 'aurora', 'beta', 'nightly']
channel_pings = []
for channel in channels:
   channel_pings.append(get_pings(sc, channel=channel, **kwargs))
pings = sc.union(channel_pings)
```

This may just be a bit easier to read, but both works.